carry forward force_policy to the Queryset

srj55 opened the pull request:
force_policy feature is currently only available on the model instances.  Enable this feature for SafeDeleteQuerySet as well.

Gagaro commented:
Hi, thanks for the PR :+1: 

Could you rebase on master, do the same for `undelete` and add the changelog in `CHANGES`?

srj55 commented:
i've rebased, and also updated CHANGES.

For undelete, the _safedelete_policy isn't implemented, so 'force_policy' wouldn't apply.  Perhaps we should add this feature to undelete in the future?

Gagaro commented:
Indeed my bad. Thanks :+1: .
